workspaces/ui-v2/src/lib/__tests/diff-helpers/universes/makeCurrentSpecContext.ts

```diff
@@ -16,11 +14,11 @@ delete global.self;
 
 export async function makeCurrentSpecContext(
   events: any[],
-  query: IBaseSpectacle['query']
```

Comment:
This is actually the wrong type signature, since we actually need the queryWrapper signature, however it's harder to extract, and this is just a test so 🤷